public/locales/az/translation.json

```diff
@@ -170,16 +170,17 @@
             "update_game": "Oyunu yeniləyin"
         }
     },
+    "disk": "Disk",
     "docs": "Sənədlər",
+    "down": "Down",
```

Comment:
this might be confusing for translators without context (like `down` in spanish would be `abajo` but in the context of downloads would be more like `download speed` / `velocidad de descarga` so `descarga` would be a better translation)

maybe `speed` is better than `down`?

Reply:
fixed